plugins/destination/file/internal/backends/azure-blob-storage/file.go

```diff
@@ -0,0 +1,124 @@
+package azure_blob_storage
```

Comment:
can we keep the directory also `azure_blog_storage` instead of `azure-blog-storage` ?

Reply:
Fixed